brainstormforce/ultimate-addons-for-gutenberg
# Fix: Forms - Toggle default color, Input default color changes.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/fbcbf600-996b-445d-a8bb-61b2e6e33222?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Forms in next-release and add input color from style > Input tab. It will apply to the field border.
- Drag and drop Toggle in forms and enable disable it. It will not work properly.
- Switch to the above branch to check if the issue is fixed or not.

### Checklist:
